portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+// comment message
+public final class Comment {
+
+    private final String name;
```

Comment:
no particular reason actually, just that it looks nicer to read in the db. will change soon. thanks!

Reply:
solved in commit bf284a4

Reply:
I see. You want readable format in DB's admin page. That's a reasonable design. 

For such purpose, one option is to use the built-in Date-Time type of the DB, in this case it would be the Date and Time type in Datastore. https://cloud.google.com/datastore/docs/concepts/entities#datastore-datastore-properties-java

Simply using epoch milliseconds is also a common option, it's just a very handy solution and usually can be converted to human-readable format in front-end code.

Reply:
yeah, i was hoping for something like that (datetime in Python). yeah, actually it's quite convenient to convert dates using unix time. thanks for the suggestion! actually, is it better to make the conversion in BE or FE? I feel like in my current website, loading comments took quite a while.

Reply:
I think as long as the database support datetime type, then using native datetime is the best option. However sometimes you have to also consider the supported type values in other modules/languages, so unix time can be a universally supported type. Timestamps are location specific, you always want to display the value in a local timezone, so usually we put a conversion in the FE, because it's the FE that can detect the user's timezone.